### DISCUSSION TITLE
Managing schema by dividing in module.

### DISCUSSION THREAD

**Discussion by ItalicJerrold** [99designs/gqlgen#2233] [Q&A]
### What happened?
We have quite a lot API in one schema and we want to separate them into different schema files so that we can better manage them. However, we found that type Query, Subscription and Mutation will cause conflict in a generation.
### What did you expect?
Can conflict check downgrade to query/mutation/subscription name but not to those special type?
### Minimal graphql.schema and models to reproduce
Create two schemas with `type Query` and run gqlgen
### versions
gqlgen:  v 0.13.0
go 1.17.8

**Comment by davidkus** [comment]
It was suggested in another issue to use `extend type Query`, which seemed to work for me. Maybe this will work for you as well? Example:

users.graphql.schema
``` 
type Query {
  user: User!
}
```

books.graphql.schema
```
extend type Query {
  book: Book!
}
```
